buidler.config.ts

```diff
@@ -0,0 +1,14 @@
+import { usePlugin } from "@nomiclabs/buidler/config";
+
+usePlugin("@nomiclabs/buidler-waffle");
+
+export default {
+  paths: {
+    artifacts: "build/artifacts",
+    cache: "build/cache",
+    sources: "src/contracts",
+  },
+  solc: {
+    version: "0.6.12",
```

Comment:
Maybe this is a good PR in which to change the contracts' pragma to `^0.6.12` ?

Reply:
I think we could do that as a separate trivial PR. Also, I'm not sure how important it is, it would make other projects that import our contracts to also require solc `^0.6.12`, even if they just want to interface with it. Since solidity enforces a common ABI and syntax across all versions `0.x.y` for the same value of `x`, I think its better to leave the contracts with the current pragma.

Reply:
Makes sense. The reason I was proposing this is that the `Math.sol` library has `pragma solidity ^0.6.12;`, so switching to that would make everything use the same version.

Reply:
Very good point - I'll do it in a follow up PR as its unrelated to these changes.